component/authenticator/src/test/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAuthenticatorTest.java

```diff
@@ -207,12 +223,17 @@ public void testProcess(boolean isLogoutRequest, String backupCode, String authe
                             boolean expectError) throws BackupCodeException, UserStoreException {
 
         BackupCodeAuthenticator backupCodeAuthenticator = new BackupCodeAuthenticator();
+        Map<String, String> parameterMap = new HashMap<>();
+        parameterMap.put(FrameworkConstants.SHOW_AUTHFAILURE_RESON_CONFIG, "false");
+        AuthenticatorConfig authenticatorConfig = new AuthenticatorConfig(
+                "backup-code-authenticator", true, parameterMap);
```

Comment:
We have already defined `backup-code-authenticator` as a constant. Shall we refer that constant here

Reply:
adressed in e518f298d3807b3fb52c908b88d6ee9be525793b